Makefile

```diff
@@ -35,44 +35,50 @@ ifdef PREBUILT_ASSETS_STATIC_DIR
   SKIP_UI_BUILD = true
 endif
 
+help: ## Displays commands and their descriptions.
```

Comment:
This is great, I only wonder if we shouldn't do this in `Makefile.common` which we propagate cross organization (via https://github.com/prometheus/prometheus/blob/main/scripts/sync_repo_files.sh).

WDYT @SuperQ ?